pkg/prom/instance/configstore/remote.go

```diff
@@ -162,6 +219,36 @@ func (r *Remote) List(ctx context.Context) ([]string, error) {
 	return r.kv.List(ctx, "")
 }
 
+// ListConsul returns Key Value Pairs instead of []string
```

Comment:
nit: Comment doesn't match function signature
```suggestion
// listConsul returns Key Value Pairs instead of []string
```